src/camera.cpp

```diff
@@ -407,8 +424,7 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime,
 	//v3f wield_rotation = v3f(-100, 120, -100);
 	v3f wield_rotation = v3f(-100, 120, -100);
 	wield_position.Y += fabs(m_wield_change_timer)*320 - 40;
-	if(m_digging_anim < 0.05 || m_digging_anim > 0.5)
-	{
+	if(m_digging_anim < 0.05 || m_digging_anim > 0.5) {
```

Comment:
Apparently it's a tool recharge animation. https://github.com/minetest/minetest/commit/989aba1966373885d4cac306daea25e102f8d68d

@celeron55, please may you explain?